### PR TITLE
khalid/ID naming

### PR DESCRIPTION
**Pull request by khalid-deriv**
Changes:

-   Added human-readable ID names to all buttons and forms

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/home/_trade-types-mobile.js [hunk 1]
```diff
@@ -167,7 +167,12 @@ const TradeTypesMobile = () => {
                         />
                     ))}
                 </Carousel>
-                <TradingButton type="submit" secondary="true" to="/signup/">
+                <TradingButton
+                    is="dm-trade-types-mobile-signup"
```
Comment (sean-binary) [inline src/pages/home/_trade-types-mobile.js:171]:
```suggestion
                    id="dm-trade-types-mobile-signup"
```

File: src/pages/p2p/components/_availability.js [hunk 2]
```diff
@@ -168,14 +168,16 @@ const Availability = () => {
...
                                         size={24}
                                         onClick={handleLogin}
                                         color="red"
                                         external
                                         key={0}
                                     />,
                                     <LocalizedLinkText
-                                        to="/signup/"
+                                        id="dm-p2p-signup-link"
+                                        to="/signup"
```
Comment (sean-binary) [inline src/pages/p2p/components/_availability.js:180]:
```suggestion
                                        to="/signup/"
```

**Review by sean-binary — CHANGES_REQUESTED** [review]

**Comment by sean-binary** [comment]
Do we ever use the existing ids for styling?

**Comment by khalid-deriv** [comment]
> Do we ever use the existing ids for styling?
No, we don't I made sure. Godbless styled components 😄 
Let me check again though just to be 100% sure

**Review by sean-binary — APPROVED** [review]

**Pull request merged**